src/v/pandaproxy/schema_registry/store.h

```diff
@@ -56,49 +56,6 @@ class store {
         return {version, id, inserted};
     }
 
-    bool version_exists(subject sub, schema_id sid) {
```

Comment:
Can the PR be reordered so that this code never existed?

Reply:
Done